code/_onclick/click.dm

```diff
@@ -253,8 +256,21 @@
 	A.CtrlShiftClick(src)
 	return
 
-/atom/proc/CtrlShiftClick(var/mob/user)
+/atom/proc/CtrlShiftClick(var/mob/living/user) 	//PybroPointClick, 1/30/15
+	if(istype(user, /mob/living/carbon/brain || /mob/living/silicon/pai)) //If this isn't in place, then they can still point. They won't actually make an arrow/emote, but they'll still get the tired finger message. Pedantic, but whatever.

```

Comment:
The istype()s you're placing Non Object Oriented Code in a proc specifically set up to be object oriented.

You should:
* Create a ```/mob/living/carbon/brain/CtrlShiftClick(var/mob/living/user)``` that just returns
* Create a ```/mob/living/silicon/pai/CtrlShiftClick(var/mob/living/user)``` that just returns